src/routes/donate/+page.svelte

```diff
@@ -206,7 +206,7 @@
                 </p>
                 <p class="small">
                     <LocalizedText
-                        text="You can always donate to our parent projects Scratch or TurboWarp as well, to help them stay online."
+                        text="You can always donate to our parent project Scratch as well, to help it stay online."
```

Comment:
marking this as review so its clear that we need to update translations for this change to work